### ISSUE TITLE
[Docs] Show how to download from GitHub in more places

### ISSUE THREAD

**Issue by DNin01** [STForScratch/ScratchTools#291]
The first target for anyone with no experience with GitHub, contributing, or browser extensions would probably be the [Installing](https://docs.scratchtools.app/docs/getting-started/installing) page from the docs, which tells you how to install ScratchTools from web stores, but web store installations aren't meant to be for modifying code, in fact, I think the browser might even throw an error if you do. The next obvious place to go to is [Building a Feature](https://docs.scratchtools.app/docs/contributing/building-a-feature), but it says nothing about running ScratchTools or how to test the modified code, and these users will want to test their changes.

**There are a few ways we could solve this:**

- [Installing Beta](https://docs.scratchtools.app/docs/beta) does go through the developer-oriented way to install the extension that you would use to test the changes you made. We could link to it from [Building a Feature](https://docs.scratchtools.app/docs/contributing/building-a-feature) to make sure the user understands how to test their changes.

- We could also add a note to [Installing](https://docs.scratchtools.app/docs/getting-started/installing) that tells you what to do if you plan to make changes or contribute, again, linking to Installing Beta, like the [How do I download ScratchTools? FAQ](https://docs.scratchtools.app/docs/faq#how-do-i-download-scratchtools) sort of does.

- We could also link to those instructions from the [CONTRIBUTING.md](https://github.com/STForScratch/ScratchTools/blob/main/CONTRIBUTING.md) file in this repository.

While we're at it, we should also note that you should be testing your changes before you submit them.

**Comment by rgantzos** [comment]
I like this. We recently changed our documentation website and information ([STForScratch/docs](https://github.com/STForScratch/docs)) and we definitely need to incorporate it into GitHub and our website better. Part of that means updating the documentation to better support new contributors/people looking to contribute.

**Comment by MaterArc** [comment]
Yeah, I agree. Many people ask about how to download the beta. This would help to inform and direct users to the right place more easily. Great idea!

**Comment by DNin01** [comment]
I'll see what I can do.
